### ISSUE TITLE
[Accessibility- Information presentation C 10.7] Visual indication of focus

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#2918]
**Environment:** https://whitepaper-ft.meeds.io/

**Step of reproduction:**
Enable the focus of left menu and top bar items by tabbing or by using pointing device

**Current behavior:**
A lot of interactive elements on the platform currently have a focus style defined but not visible enough (see attachments bellow)

**Expected behavior:**
Use the native browser focus

![Image](https://github.com/user-attachments/assets/96906f8f-bc02-4730-befc-72c127c23454)

![Image](https://github.com/user-attachments/assets/6f7dff78-9923-481d-a79f-6a7cebbf6b20)

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
